config/initializers/extend_module.rb

```diff
@@ -0,0 +1,7 @@
+class Module
+  def extend_each_subclass(inklude)
+    constants.each do |sym|
```

Comment:
@lindgrenj6 I was hoping we could use `descendants` here, but that does not seem to work.  Are we sure this is a safe way to enumerate the classes?

Reply:
Well the only way we'd be able to use `descendants` would be to iterate over each v1x0 controller (that we haven't overriden with new v1x1 behavior) and iterate over the descendants again to send the `:include` message. 
The quickest way to do that would probably be to iterate over the constants again to get that list unless we had a hardcoded list. Which would be a bit off because we'd then have the class definitions + a list of them.

After digging in a bit, I do think that `constants` is a safe way to enumerate over the controllers since each class (that we're just inheriting from v1x0 anyway) is just a constant on the module. I could add some filtering to only extend on things that end with `Controller` as well to make it safer maybe?

Is there something I'm missing here?

Reply:
Right, I looked at little more as well.  I think we are good here.  Thanks for checking.